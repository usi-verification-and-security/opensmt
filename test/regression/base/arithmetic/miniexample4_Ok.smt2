(set-logic QF_LRA)
(declare-fun x () Real)
(declare-fun y () Real)
(define-fun uninterp_div ((a Real) (b Real)) Real (/ a b))
(assert (= (uninterp_div y 5) (+ 3 x)))
(check-sat)
(get-model)
(exit)
