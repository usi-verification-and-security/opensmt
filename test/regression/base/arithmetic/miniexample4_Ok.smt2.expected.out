sat
(
  (define-fun x () Real
    0)
  (define-fun y () Real
    15)
)
